src/com/yetanalytics/lrs_admin_ui/language.cljs

```diff
@@ -0,0 +1,185 @@
+(ns com.yetanalytics.lrs-admin-ui.language)
+
+(def language
+  {;;Footer   
+   :footer.attribution     {:en-US "Yet Analytics Inc."}
```

Comment:
Was there a reason why you don't have the keywords namespaced like `:footer/attribution`? (I actually kind of forgot that's still a valid keyword)

Reply:
they get very complex and there's not a good definition of what is the important "namespace" part, plus we're not really dealing with a code namespace either, AND they're being converted from (and to) JSON.